app/views/layouts/spina/admin/admin.html.erb

```diff
@@ -13,6 +13,8 @@
   
   <!-- Make sure all modals are rendered in this frame -->
   <%= turbo_frame_tag "modal" %>
+
+  <%= render Spina::Forms::AutoFileUploadComponent.new %>
```

Comment:
This sits at the bottom of every page like the `modal` section and just listens for the `auto-file-upload:start` event to do its thing. It takes the file which should be included in that event, puts it into the file component form, and automates a Turbo request to ImagesController.

Reply:
Is it necessary to put this in the layout? Now it's also rendered on pages without Trix where it's not being used.